### PR TITLE
PR Committee Scraper

### PR DESCRIPTION
**Pull request by jgibson517**
I left all the committee names and roles in Spanish, didn't think it made sense to change them! Let me know if there are any questions.

### CODE REVIEW

File: scrapers_next/pr/committees.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    def process_page(self):
+        com = self.input
+        com.add_source(self.source.url, "Committee Detail Page")
+
+        members = CSS("div.panel-body div.senator_cont a").match(self.root)
+
+        for member in members:
+            name = CSS("span.name").match_one(member).text_content()
+            role = CSS("span.position").match_one(member).text_content()
+            if role == "":
```
Comment (johnseekins) [inline scrapers_next/pr/committees.py:20]: `if not role` is a bit broader. Is there a reason we're doing `if role == ""`?

File: scrapers_next/pr/committees.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    example_source = "https://www.camara.pr.gov/ova_dep/comision-conjunta-para-la-revision-y-reforma-del-codigo-civil-de-puerto-rico/"
+
+    def process_page(self):
+        com = self.input
+        com.add_source(self.source.url, note="Committee Detail Page")
+
+        # Some House committes have different formats
+        try:
+            members = CSS("div.mc-title-container").match(self.root)
+            for member in members:
```
Comment (johnseekins) [inline scrapers_next/pr/committees.py:38]: If possible, let's keep `for` loops outside `try` blocks.

File: scrapers_next/pr/committees.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                if role == "":
+                    role = "member"
+
+                com.add_member(name, role)
+        except SelectorError:
+            pass
+
+        try:
+            members = CSS("div.elementor-widget-container ul li.p1").match(self.root)
+            for member in members:
```
Comment (johnseekins) [inline scrapers_next/pr/committees.py:51]: As above, let's keep the `for` loop out of the `try` block.

File: scrapers_next/pr/committees.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                    "Presidente|Secretari[ao]|Vice-presidente", member.text_content()
+                )
+                name = re.sub("Hon. |,.*", "", member.text_content())
+                if title != []:
+                    role = title[0]
+                com.add_member(name, role)
+        except SelectorError:
+            pass
+
+        return com
```
Comment (johnseekins) [inline scrapers_next/pr/committees.py:63]: Worth looking for an empty committee list and returning an EmptyScrape object?

File: scrapers_next/pr/committees.py (hunk 1 above)
Comment (jgibson517) [reply scrapers_next/pr/committees.py:20]: Changed this and restructured the try statements in most recent commit

**Review by jgibson517 — COMMENTED** [review]
Reformatted try statements.

**Review by chrisyamas — APPROVED** [review]
Amazing work, @jgibson517 ! Only made some minor modifications, and it's ready to merge now.

**Pull request merged**